src/PrestaShopBundle/Resources/config/services/core/domain/cart.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  _defaults:
+    autowire: true
+    public: false
+
+  PrestaShop\PrestaShop\Adapter\Cart\Repository\CartRepository:
```

Comment:
oh this should be defined in adapter/cart.yml as you can see by its namespace (its not in domain). So you can get rid of this file afterwards.